src/lib.rs

```diff
@@ -0,0 +1,408 @@
+                    source,
+                    &target,
+                    rev,
+                    &mut config,
+                )?)
+            }
+            RustdocSource::Version(version) => {
+                let mut registry = {
+                    let manifest_dir = match &self.current.source {
+                        RustdocSource::Root(manifest_dir)
+                        | RustdocSource::Revision(manifest_dir, _) => manifest_dir,
+                        RustdocSource::Version(_) | RustdocSource::Rustdoc(_) => {
+                            anyhow::bail!("this combination of current and baseline sources isn't supported yet")
+                        }
+                    };
+                    let metadata = manifest_metadata_no_deps(manifest_dir)?;
+                    let target = metadata.target_directory.as_std_path().join(util::SCOPE);
+                    baseline::RegistryBaseline::new(&target, &mut config)?
+                };
+                if let Some(ver) = version {
+                    let semver = semver::Version::parse(ver)?;
+                    registry.set_version(semver);
+                }
+                Box::new(registry)
+            }
+        };
+        let rustdoc_cmd = dump::RustDocCommand::new()
+            .deps(false)
+            .silence(!config.is_verbose());
+
+        let all_outcomes: Vec<anyhow::Result<bool>> = match &self.current.source {
+            RustdocSource::Rustdoc(current_rustdoc_path) => {
+                let name = "<unknown>";
+                let version = None;
+                let (current_crate, baseline_crate) = generate_versioned_crates(
+                    &mut config,
+                    CurrentCratePath::CurrentRustdocPath(current_rustdoc_path),
+                    &*loader,
+                    &rustdoc_cmd,
+                    name,
+                    version,
+                )?;
+
+                let success = run_check_release(&mut config, name, current_crate, baseline_crate)?;
+                vec![Ok(success)]
+            }
+            RustdocSource::Root(project_root) => {
+                let metadata = manifest_metadata(project_root)?;
+                let selected = self.scope.selected_packages(&metadata);
+                selected
+                    .iter()
+                    .map(|selected| {
+                        let manifest_path = selected.manifest_path.as_std_path();
+                        let crate_name = &selected.name;
+                        let version = &selected.version;
+
+                        let is_implied = matches!(
+                            self.scope.mode,
+                            ScopeMode::DenyList(PackageSelection {
+                                selection: ScopeSelection::Workspace,
+                                ..
+                            })
+                        ) && selected.publish == Some(vec![]);
+                        if is_implied && selected.publish == Some(vec![]) {
```

Comment:
One of the `selected.publish == ...` is not needed.

Reply:
fixed in [371ccab](https://github.com/obi1kenobi/cargo-semver-checks/pull/314/commits/371ccabd22e9253d6aa0ef85edd7774fbe327f42)